games/o.yaml

```diff
@@ -1552,6 +1552,28 @@
   type: remake
   updated: 2015-04-04
 
+- name: OpenSupaplex
+  lang:
+  - C
+  framework:
+  - SDL2
+  license:
+  - GPL3
```

Comment:
```suggestion
  - GPL3
  content: free
```

Reply:
Done, thanks for the suggestion!

I've also changed the type from `clone` to `remake`, to match other remakes I've seen that are actually reimplementations 😄